bootstrap.py

```diff
@@ -397,7 +397,12 @@ def disassociate_host(host_id):
 
 
 def check_rhn_registration():
-    return os.path.exists('/etc/sysconfig/rhn/systemid')
+    if os.path.exists('/etc/sysconfig/rhn/systemid'):
+        output = commands.getstatusoutput('rhn-channel -l')
+        retcode = output[0]
+        return retcode == 0
```

Comment:
can't we simply put that on a single line?
```python
return commands.getstatusoutput('rhn-channel -l')[0] == 0
```

Just asking, not sure what's our priority. Not a blocker